libraries/rush-lib/src/utilities/InteractiveUpgradeUI.ts

```diff
@@ -0,0 +1,187 @@
+import colors from 'colors/safe';
+import inquirer from 'inquirer';
+import type * as inquirerTypes from 'inquirer';
+import table from 'text-table';
+import _ from 'lodash';
```

Comment:
I have a "kill it with fire" relationship with `lodash` dependencies since most things it does are natively supported in ECMAScript nowadays.